app/controllers/api/v1/people_controller.rb

```diff
@@ -247,7 +247,13 @@ def request_password_reset
     require_params(:email)
 
     if (person = Person.find_by_email(params[:email]))
-      person.send_email(:reset_password)
+      if URI(request.referer).host == URI.parse(Api::Application.config.salt_url).host
```

Comment:
I don't think `request.referer` is reliable... could we instead pass a `is_salt=true` boolean as a parameter?